src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -8,6 +8,12 @@ namespace CommunityToolkit.Maui.Core;
 /// </summary>
 public interface IMediaElement : IView
 {
+	/// <summary>
+	/// Gets or sets the media aspect.
+	/// </summary>
+	/// <remarks>Not functional for non-visual media.</remarks>
+	Aspect Aspect { get; set; }
```

Comment:
If the Platform layer will not `set` the aspect, it should be readonly, from what I saw in the code... The platform layer just reads this value and applies it. If I missed something, please let me know

```suggestion
	Aspect Aspect { get; }
```